google/cloud/storage/examples/storage_examples_common.cc

```diff
@@ -97,7 +98,7 @@ Status RemoveBucketAndContents(google::cloud::storage::Client client,
 Status RemoveStaleBuckets(
     google::cloud::storage::Client client, std::string const& prefix,
     std::chrono::system_clock::time_point created_time_limit) {
-  std::regex re("^" + prefix + R"re(\d{4}-\d{2}-\d{2}_.*$)re");
+  std::regex re("^" + prefix + R"re(_\d{4}-\d{2}-\d{2}_.*$)re");
```

Comment:
Do we still need/want the `prefix` here?

Reply:
I think so?  Maybe we can revisit when I refactor the `RemoveStaleBuckets()` function.